nix/workbench/backend/supervisor.nix

```diff
@@ -15,7 +15,7 @@ let
     ]
   ++ lib.optionals ( useCabalRun)
     (with haskellPackages; [
-      cabalWrapped
+      cabal-install
```

Comment:
You should now also drop the explicit `packages` argument in the `haskell.nix` shell function.

Reply:
Oh, yes, right, forgot about that one.

Thank you!